src/app/archive/archive_lib/processor.ml

```diff
@@ -476,9 +489,7 @@ module Block = struct
         (* For technique reasons, there might be multiple fee transfers for
            one receiver. As suggested by deepthi, I combine all the fee transfer
            that goes to one public key here *)
-        let fee_transfer_table =
-          Core.Hashtbl.create (module Signature_lib.Public_key.Compressed)
-        in
+        let fee_transfer_table = Core.Hashtbl.create (module Account_id) in
         let () =
           let open Coda_base in
           Core.List.iter fee_transfers ~f:(fun fee_transfer ->
```

Comment:
I think every `Core` in this file could be `Core_kernel`. Not a big deal, because the dune imports `core`.